packages/readline/readline.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for GNU readline"
+maintainer: ["vincent.tourneur@inria.fr"]
+authors: ["Sylvain Pogodalla" "Vincent Tourneur"]
+license: "CeCILL-2.0+"
+homepage: "https://gitlab.inria.fr/vtourneu/readline-ocaml"
+bug-reports: "https://gitlab.inria.fr/vtourneu/readline-ocaml/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "ocaml" {>= "4.14"}
+  "conf-readline" {>= "0.0"}
```

Comment:
```suggestion
  "conf-readline"
```
Let's merge this change once the `conf-readline` package from the other PR is merged, so that we can test it